packages/arb-token-bridge-ui/src/components/common/TestnetToggle.tsx

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import { useIsTestnetMode } from '../../hooks/useIsTestnetMode'
+
+import { Switch } from './atoms/Switch'
+import { warningToast } from './atoms/Toast'
+import { twMerge } from 'tailwind-merge'
+
+export const TestnetToggle = ({
+  className,
+  label,
+  description
+}: {
+  className?: {
+    wrapper?: string
+    switch?: string
+  }
+  label: string
+  description?: string
+}) => {
+  const { isSourceChainTestnet, isTestnetMode, setIsTestnetMode } =
+    useIsTestnetMode()
+
+  const enableTestnetMode = useCallback(() => {
+    setIsTestnetMode(true)
+  }, [setIsTestnetMode])
+
+  const disableTestnetMode = useCallback(() => {
```

Comment:
As discussed, let's let users to disable the testnet mode regardless of the source network, and default to Ethereum if the source network was testnet.